src/validate_i18n.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import re
+
+from flask import request
+from werkzeug.exceptions import abort
```

Comment:
Was it a conscious decision not to use the abort method from the flask module?

Reply:
No, it was just a mistake due to overeager IDE imports. Have changed it back.